packages/superset-ui-plugin-chart-icicle-event/src/IcicleEventChart.tsx

```diff
@@ -29,14 +31,41 @@ interface Props {
     y: number;
   };
   color: (name: string) => string;
-  contentRenderer: () => void;
+  contentRenderer: (...args: any[]) => void;
```

Comment:
Can we more more explicit on the type?

Reply:
Updated to include the types, `BaseType` is documented here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-selection/index.d.ts#L21